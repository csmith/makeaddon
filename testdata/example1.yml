package-as: SomeAddon

externals:
  libs/AceAddon-3.0: https://repos.wowace.com/wow/ace3/trunk/AceAddon-3.0
  libs/AceDB-3.0: https://repos.wowace.com/wow/ace3/trunk/AceDB-3.0
  libs/AceDBOptions-3.0: https://repos.wowace.com/wow/ace3/trunk/AceDBOptions-3.0
  libs/AceHook-3.0: https://repos.wowace.com/wow/ace3/trunk/AceHook-3.0
  libs/AceGUI-3.0: https://repos.wowace.com/wow/ace3/trunk/AceGUI-3.0
  libs/AceConsole-3.0: https://repos.wowace.com/wow/ace3/trunk/AceConsole-3.0
  libs/AceConfig-3.0: https://repos.wowace.com/wow/ace3/trunk/AceConfig-3.0
  libs/LibDataBroker-1.1: https://github.com/tekkub/libdatabroker-1-1

move-folders:
  SomeAddon/SomeAddon_Config: SomeAddon_Config

ignore:
  - tests
  - deploy
