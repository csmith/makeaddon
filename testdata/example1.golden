{
  "PackageAs": "SomeAddon",
  "ManualChangelog": null,
  "Externals": {
    "libs/AceAddon-3.0": {
      "Url": "https://repos.wowace.com/wow/ace3/trunk/AceAddon-3.0",
      "Tag": ""
    },
    "libs/AceConfig-3.0": {
      "Url": "https://repos.wowace.com/wow/ace3/trunk/AceConfig-3.0",
      "Tag": ""
    },
    "libs/AceConsole-3.0": {
      "Url": "https://repos.wowace.com/wow/ace3/trunk/AceConsole-3.0",
      "Tag": ""
    },
    "libs/AceDB-3.0": {
      "Url": "https://repos.wowace.com/wow/ace3/trunk/AceDB-3.0",
      "Tag": ""
    },
    "libs/AceDBOptions-3.0": {
      "Url": "https://repos.wowace.com/wow/ace3/trunk/AceDBOptions-3.0",
      "Tag": ""
    },
    "libs/AceGUI-3.0": {
      "Url": "https://repos.wowace.com/wow/ace3/trunk/AceGUI-3.0",
      "Tag": ""
    },
    "libs/AceHook-3.0": {
      "Url": "https://repos.wowace.com/wow/ace3/trunk/AceHook-3.0",
      "Tag": ""
    },
    "libs/LibDataBroker-1.1": {
      "Url": "https://github.com/tekkub/libdatabroker-1-1",
      "Tag": ""
    }
  },
  "MoveFolders": {
    "SomeAddon/SomeAddon_Config": "SomeAddon_Config"
  },
  "Ignore": [
    "tests",
    "deploy"
  ],
  "LicenseOutput": "",
  "NoLibCreation": true
}